<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomReminder</class>
 <widget class="QMainWindow" name="CustomReminder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>自定义提醒</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>382</width>
        <height>270</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="StartPage">
       <widget class="QComboBox" name="StartComboBox">
        <property name="geometry">
         <rect>
          <x>88</x>
          <y>70</y>
          <width>200</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QPushButton" name="StartButton">
        <property name="geometry">
         <rect>
          <x>143</x>
          <y>140</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="PresetsPage">
       <widget class="QComboBox" name="PresetsComboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>176</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="PresetsLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>30</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>预设</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="TitleLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>60</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>标题</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="TextLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>90</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>文本</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="TitleLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>176</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="TextLineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>176</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="NewButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>新建预设</string>
        </property>
       </widget>
       <widget class="QPushButton" name="DeleteButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>删除预设</string>
        </property>
       </widget>
       <widget class="QPushButton" name="TestButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>测试提醒</string>
        </property>
       </widget>
       <widget class="QLabel" name="TimeLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>120</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="HourLabel">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>120</y>
          <width>16</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>时</string>
        </property>
       </widget>
       <widget class="QLabel" name="MinuteLabel">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>120</y>
          <width>16</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>分</string>
        </property>
       </widget>
       <widget class="QLabel" name="SecondLabel">
        <property name="geometry">
         <rect>
          <x>225</x>
          <y>120</y>
          <width>16</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>秒</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="HourSpinBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>42</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="MinuteSpinBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>42</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="SecondSpinBox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>42</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="SoundComboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>150</y>
          <width>176</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="PlayModeComboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>176</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="RepeatComboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>210</y>
          <width>67</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="MiddleTimeComboBox">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>210</y>
          <width>46</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="SoundLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>150</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>声音</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="PlayModeLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>180</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>播放模式</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="RepeatLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>210</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>重复</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="MiddleTimeLabel">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>210</y>
          <width>50</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>间隔时间</string>
        </property>
       </widget>
       <widget class="QLabel" name="MiddleMinuteLabel">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>210</y>
          <width>26</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>分钟</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SaveButton">
        <property name="geometry">
         <rect>
          <x>305</x>
          <y>230</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>保存预设</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="SettingsPage">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>97</width>
          <height>48</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="MinCheckBox">
           <property name="text">
            <string>最小化到托盘</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="TopCheckBox">
           <property name="text">
            <string>置顶</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="TimerPage">
       <widget class="QLabel" name="TimerLabel">
        <property name="geometry">
         <rect>
          <x>-2</x>
          <y>25</y>
          <width>386</width>
          <height>136</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(70, 70, 70);</string>
        </property>
        <property name="text">
         <string>Timer</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="ContinueButton">
        <property name="geometry">
         <rect>
          <x>35</x>
          <y>200</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>继续</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StopButton">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>200</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
       <widget class="QPushButton" name="EndButton">
        <property name="geometry">
         <rect>
          <x>255</x>
          <y>200</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>结束</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CustomReminder.qrc"/>
 </resources>
 <connections/>
</ui>
